packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -651,6 +651,46 @@ const ReleaseDetailsBody = ({ releaseId }: ReleaseDetailsBodyProps) => {
     );
   }
 
+  const headers = [
+    {
+      label: formatMessage({
+        id: 'content-releases.page.ReleaseDetails.table.header.label.name',
+        defaultMessage: 'name',
+      }),
+      name: 'name',
+    },
+    {
+      label: formatMessage({
+        id: 'content-releases.page.ReleaseDetails.table.header.label.locale',
+        defaultMessage: 'locale',
+      }),
+      name: 'locale',
+    },
+    {
+      label: formatMessage({
+        id: 'content-releases.page.ReleaseDetails.table.header.label.content-type',
+        defaultMessage: 'content-type',
+      }),
+      name: 'content-type',
+    },
+    {
+      label: formatMessage({
+        id: 'content-releases.page.ReleaseDetails.table.header.label.action',
+        defaultMessage: 'action',
+      }),
+      name: 'action',
+    },
+    !release.releasedAt
+      ? {
+          label: formatMessage({
+            id: 'content-releases.page.ReleaseDetails.table.header.label.status',
+            defaultMessage: 'status',
+          }),
+          name: 'status',
+        }
+      : null,
+  ].filter((head) => Boolean(head)) as Table.Header<FormattedReleaseAction, any>[];
```

Comment:
do you think that reads better? I'm always so on the fence about this 🤔

Reply:
For me the readability is about the same, the filter just seems a bit much for 1 item that could be null, but I'm not opposed to what you have here either since technically the list could grow.

Reply:
i think your way is probably better because i have to cast it 🙃 

i just want a native way to optionally declare items in an array.... pls javascript